src/map/clif.c

```diff
@@ -9338,7 +9341,7 @@ void clif_parse_LoadEndAck(int fd, struct map_session_data *sd) {
 	sd->state.warping = 0;
 	sd->state.dialog = 0;/* reset when warping, client dialog will go missing */
 
-	// look
+	//< Character Looks
```

Comment:
But if it recognizes `//`, and it should, anything after it would be treated as a comment no?